src/styles/index.scss

```diff
@@ -0,0 +1,9 @@
+@import "./utils/grid";
```

Comment:
firstly import variables here, they are used in nearly every file
then no need to import them to every scss block, just here